nbqa/cmdline.py

```diff
@@ -1,14 +1,34 @@
 """Parses the command line arguments provided."""
 import argparse
-import sys
 from textwrap import dedent
 from typing import List, Optional
 
 from nbqa import __version__
 
-RED = "\x1b[1;31m"
+BOLD = "\033[1m"
 RESET = "\x1b[0m"
 CONFIGURATION_URL = "https://nbqa.readthedocs.io/en/latest/configuration.html"
+DOCS_URL = "https://nbqa.readthedocs.io/en/latest/index.html"
+USAGE_MSG = dedent(
+    f"""\
+    nbqa <code quality tool> <notebook or directory> <nbqa options> <code quality tool arguments>
+
+    {BOLD}Please specify:{RESET}
+    - 1) a code quality tool (e.g. `black`, `pyupgrade`, `flake`, ...)
+    - 2) some notebooks (or, if supported by the tool, directories)
+    - 3) (optional) extra flags
```

Comment:
Do you need to correct these to `3) (optional) extra flags`?

Reply:
not sure I understand, sorry

Reply:
Since you have changed to `nbqa <code quality tool> <notebook or directory> <nbqa options> <code quality tool arguments>`, do you think `3) (optional) extra flags` should also be changed to something like `3) Options to pass to nbqa (ex: --nbqa-mutate)`, `4) Arguments to pass to the code quality tool`

Reply:
I have approved the PR anyway, just thought of asking you about this.

Reply:
yeah you're right thanks!